src/app/pages/loading-waiting-states/contextual-spinners/contextual-spinners.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit } from '@angular/core';
+import { BreakpointObserver, Breakpoints, BreakpointState } from '@angular/cdk/layout';
+import { MatSlideToggleChange } from '@angular/material/slide-toggle';
+import { StateService } from '../../../services/state.service';
+
+@Component({
+    selector: 'app-contextual-spinners',
+    templateUrl: './contextual-spinners.component.html',
+    styleUrls: ['./contextual-spinners.component.scss'],
+})
+export class ContextualSpinnersComponent implements OnInit {
+    isSmall: boolean;
+    checked = false;
+    buttonClicked = false;
+    flatbuttonClicked = false;
```

Comment:
This is really minor, but can you update this variable to `flatButtonClicked` throughout this example to maintain the camelcase format.